### ISSUE TITLE
Button cards don't look like any other buttons in front end

### ISSUE THREAD

**Issue by mdegat01** [home-assistant/frontend#12271]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I made a button card. It looks like this:
![Screen Shot 2022-04-08 at 12 50 21 PM](https://user-images.githubusercontent.com/2037026/162486783-56f4a62b-0ded-467f-a278-268f948a5bf6.png)

Notice how different that looks from the buttons at the bottom of this pop-up:
1) White instead of blue
2) Different font-size, letter-spacing and font-weight
3) Not text-transform to uppercase
4) Does not "glow" on hover to indicate action area

### Describe the behavior you expected

I would expect this to look and feel consistent with the Home Assistant UI. It should look and function exactly like our other buttons.

### Steps to reproduce the issue

1. Add a button card to the UI
2. Compare the look and feel with the buttons shown elsewhere in the editing UI

### What version of Home Assistant Core has the issue?

core-2022.4.1

### In which browser are you experiencing the issue with?

Firefox 98.02

### Which operating system are you using to run this browser?

macOS Monterey (12.2)

### Problem-relevant frontend configuration

```yaml
show_name: true
show_icon: false
type: button
name: Test
```

**Comment by spacegaier** [comment]
I strongly disagree: It is not a regular "button", but a "button card", so it does not (and should not) have to follow how a regular button looks like. A regular button is a "dumb" element, e.g. not short press vs long press support (compared to our cards).

A regular button look would also not fit at all with the look of our cards and dashboards in general.

The action is indicated by the mouse cursor changing (as with all other interactive elements through out UI).

FYI: If you want a "real" button there is a button row in the "Entities" card. Apart from that custom elements would be the way to go to achieve what you seem to looking for.

**Comment by mdegat01** [comment]
I did forget about button row. I tested it out now but the problem with that one is there doesn't seem to be any way to hide the name and icon and just show the button. Otherwise that would work well, I was actually trying to make a button that called a script in this case.

Anyway it sounds like there was good reasons for the design differences so I'm fine with closing it. It just seemed odd to me but I see your point. I suppose most people probably use it with the icon anyway. I was basically making a mini-form in a dashboard so I wanted the form-style button but I'm sure that's an edge case.
